### ISSUE TITLE
Python 3.5.1 cannot be installed and Python 3.4 is only on ubuntu-16.04

### ISSUE THREAD

**Issue by terencehonles** [actions/setup-python#187]
**Describe the bug**

- There is no installable version of Python 3.5.1. I was trying to test against that version and noticed it was not available. In the documentation it suggested that missing versions should be requested here. Python 3.5.1 is a version of Python with partial typing support and it was needed to convert dropbox/pyannotate from Travis to GitHub actions.
- It would be nice if Python versions that don't exist on a specific version of Linux might suggest which other OSs do support that version of Python.

**Which version of the action are you using?**

- [ ] `v1`
- [x] `v2`
- [ ] Some other tag (such as `v2.0.1` or `master`)

**Environment**
- [ ] self-hosted
- [x] Linux
- [ ] Windows
- [ ] Mac

If applicable, please specify if you're using a container

**Python Versions**
Please list all of the effected versions of Python (`3.8.2`, etc.)

- 3.5.1
- 3.4.*

**Comment by funkyfuture** [comment]
Python 3.4 is only available for Ubuntu 18.04 and for those who are looking for a workaround:

```yaml
    strategy:
      matrix:
        os: [ubuntu-latest]
        python-version:
          - 2.7
          - pypy2
          - 3.5
          - 3.6
          - 3.7
          - 3.8
          - 3.9
          - pypy3
        include:
          - os: ubuntu-18.04
            python-version: 3.4
```

**Comment by MaksimZhukov** [comment]
Hello everyone! 
In accordance with the [support policy](https://github.com/actions/python-versions#support-policy), we do not provide Python versions that have reached [end-of-life](https://devguide.python.org/versions/). Please note that the Ubuntu 18.04 runner image is being deprecated. See the [related announcement](https://github.com/actions/runner-images/issues/6002) for details.
I am closing the issue. Please contact us if you have any concerns.
